### DISCUSSION TITLE
Don't know how to run the project!

### DISCUSSION THREAD

**Discussion by rafikfartas** [aub-mind/arabert#144] [Q&A]
Hello, thank you for the project, the thing is i'm new to developpement and i don't know how to u use the project for exemple in a web interface or something produced to a user, ill be gratful if u help me.
Thank you

**Comment by WissamAntoun** [comment]
Hey Rafik,

Although we don't provide examples on how to deploy or serve the models, you can still check the examples folder https://github.com/aub-mind/arabert/tree/master/examples which might help you train or experiment with some models.

I also think it is helpful to check the Arabic NLP app on huggingface https://huggingface.co/spaces/aubmindlab/Arabic-NLP to see how you leverage the pipeline class to do inference.
